back-end/src/controllers/CategoriaController.js

```diff
@@ -2,7 +2,7 @@ const CategoriaModel = require("../models/CategoriaModel");
 
 module.exports = {
   async getAll() {
-    const [_req, res] = arguments;
```

Comment:
Usar o underline no início de parâmetros de uma função é como se fosse um "acordo" de boas práticas também.

Reply:
https://cursos.alura.com.br/forum/topico-uso-do-underline-no-inicio-do-nome-de-uma-funcao-em-javascript-69334

Reply:
https://developer.mozilla.org/pt-BR/docs/Learn/JavaScript/First_steps/Variables#:~:text=N%C3%A3o%20use%20underline%20no%20in%C3%ADcio,in%C3%ADcio%20do%20nome%20de%20vari%C3%A1veis.